### PR TITLE
Add 70:20:10 to CAPP Agile Learning heading 

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Adds 70:20:10 to heading (as it's an important keyword which wasn't mentioned before in the page at all), makes the style more like the flyers and other stuff and a adds a slightly more subtle quote below the laptop image.

### CODE REVIEW

**Comment by sn3p** [comment]
Screenshots would be nice 📷

**Comment by sn3p** [comment]
Dope! Maybe we can make the slogan(s) bigger for large screens?

screen shot 2017-04-20 at 15 36 15

File: locales/nl.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ nl:
     name: CAPP Agile Learning
     desc: "Performance support door 'kennis-on-demand'. Verbeter prestaties door medewerkers te voorzien van just-in-time kennis. Leg kennis simpel vast en deel handleidingen, protocollen, werkinstructies, QR-codes e.d."
     logo: logos/capp-agile-white.png
-    heading: "Performance support voor elke organisatie."
+    heading: "Performance support voor elke organisatie.<br>70:20:10 in de praktijk."
```
Comment (sn3p) [inline locales/nl.yml]: I'm not a fan of combing slogans like this, but ok.

File: source/localizable/_hero_capp-agile.erb [hunk 2]
```diff
@@ -4,7 +4,11 @@
 
       <%= image_tag "logos/capp-agile.svg", alt: "CAPP Agile logo", class: "logo" %>
 
-      <h1><%= t("capp-agile.heading") %></h1>
+      <h1>
+        <em>
+          <%= t("capp-agile.heading") %>
```
Comment (sn3p) [inline source/localizable/_hero_capp-agile.erb]: We should avoid using [`em`](https://developer.mozilla.org/en/docs/Web/HTML/Element/em) for styling.

> Typically this element is displayed in italic type. However, it should not be used simply to apply italic styling; use the CSS styling for that purpose.

**Comment by RonaldDijkstra** [comment]
How is your green so much more green than mine? :)

File: source/localizable/_hero_capp-agile.erb (hunk 2 above)
Comment (RonaldDijkstra) [inline source/localizable/_hero_capp-agile.erb]: D'accord

**Comment by sn3p** [comment]
> How is your green so much more green than mine? :)

Weird, no difference on my screen :o

**Comment by RonaldDijkstra** [comment]
Will find another way for this. Bye bye! 👍

**Pull request closed without merging**